app/javascript/components/supplemental_application/sections/DemographicsInputs.js

```diff
@@ -11,44 +11,57 @@ const DemographicsInputs = () => {
 
   return (
     <>
-      <FormGrid.Row paddingBottom>
+      <FormGrid.Row>
         <FormGrid.Item>
           <FormGrid.Group>
-            <SelectField id='demographics-dependents'
+            <SelectField
+              id='demographics-dependents'
               fieldName='number_of_dependents'
               options={numberOfPeopleOptions}
               format={formUtils.formatNumber}
-              label='Number of Dependents' />
+              label='Number of Dependents'
+            />
           </FormGrid.Group>
         </FormGrid.Item>
-
+      </FormGrid.Row>
```

Comment:
So I'm happy with the `Object.SpecificComponent` naming convention for something that's used less frequently, but it might be nice if we started just using these directly as `<Row />` `<Group />` etc. Since they're supposed to be very generic components I think it's okay that they have short, generic names.

Something to think about, not asking you to make changes here since it's the existing pattern.